src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -154,8 +154,8 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin, EntityFilter
    * @returns The appropriate Corporation or Person name.
    */
   private formatName (filing: any): string {
-    return filing.orgName ? filing.orgName
-      : `${filing.firstName} ${filing.middleName || ''} ${filing.lastName}`
+    return filing.person.orgName ? filing.person.orgName
```

Comment:
For safety, you should check that filing isn't null and filing.person isn't null.
You may be able to use Optional Chaining (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining).

Reply:
For sure, will update!